crates/algokit_transact/src/test_utils/mod.rs

```diff
@@ -0,0 +1,330 @@
+use std::fs::File;
+
+use crate::{
+    transactions::{AssetTransferTransactionBuilder, PaymentTransactionBuilder},
+    Address, AlgorandMsgpack, Byte32, SignedTransaction, Transaction, TransactionHeaderBuilder,
+    TransactionId, TransactionType,
+};
+use base64::{prelude::BASE64_STANDARD, Engine};
+use convert_case::{Case, Casing};
+use ed25519_dalek::{Signer, SigningKey};
+use serde::Serialize;
+use serde_json::to_writer_pretty;
+
+pub struct TransactionHeaderMother {}
+impl TransactionHeaderMother {
+    pub fn testnet() -> TransactionHeaderBuilder {
```

Comment:
I think `testnet` and `mainnet` would be useful methods to have on the builder itself

Reply:
Yeah I was tossing up between here vs the builder.
The 2 reason I chose the object mother (at least for now):
- It's a behaviour that wouldn't be available in the FFI layer, so is more consistent and we wouldn't be using this in the utils libraries.
- If testnet ever needed to be reset (hopefully never), it would require a core lib update.

Ultimately it felt like this capability was an abstraction level up from the core.